lib/widgets/post/post_info_section.dart

```diff
@@ -23,105 +23,113 @@ class PostInfoSection extends StatelessWidget {
       final instanceHost = store.postView.instanceHost;
       final theme = Theme.of(context);
 
-      return Column(
-        children: [
-          Padding(
-            padding: const EdgeInsets.all(10),
-            child: Row(
-              children: [
-                if (post.community.icon != null)
-                  Padding(
-                    padding: const EdgeInsets.only(right: 10),
-                    child: InkWell(
-                      borderRadius: BorderRadius.circular(20),
-                      onTap: () => goToCommunity.byId(
-                          context, instanceHost, post.community.id),
-                      child: Avatar(
-                        url: post.community.icon,
-                        noBlank: true,
-                        radius: 20,
+      return Padding(
+        padding: const EdgeInsets.all(10),
+        child: Row(
+          children: [
+            if (post.community.icon != null)
+              Padding(
+                padding: const EdgeInsets.only(right: 10),
+                child: InkWell(
+                  borderRadius: BorderRadius.circular(20),
+                  onTap: () => goToCommunity.byId(
+                      context, instanceHost, post.community.id),
+                  child: Avatar(
+                    url: post.community.icon,
+                    noBlank: true,
+                    radius: 20,
+                  ),
+                ),
+              ),
```

Comment:
padding is displayed when it shouldn't. when hide avatars is set to true.
![telegram-cloud-photo-size-4-6048701244840720195-x](https://user-images.githubusercontent.com/10037914/139964508-1795b245-fe44-43b9-b394-813c58ee4a32.jpg)